UnitWorld_server/Main.cpp

```diff
@@ -14,13 +14,14 @@ int main()
     Logger::registerInfo([](const std::string& message) {
         std::cout << message;
     });
-    ClientRoom clientRoom(2);
+
+    ServerGame serverGame(5);
     Logger::info("Waiting for connections...\n");
-    ServerConnector serverConnector(
-        ConnectionInfo("127.0.0.1", "52124"),
-        [&clientRoom](std::shared_ptr<CommunicationHandler> communicationHandler) {
-        clientRoom.addClient(communicationHandler);
-    }
+    ServerConnector serverConnector(ConnectionInfo("127.0.0.1", "52124"),
+        [&serverGame](std::shared_ptr<CommunicationHandler> communicationHandler) {
+            Logger::info("Connection from " + communicationHandler->prettyName());
```

Comment:
doesn't the Minecraft server also logs messages to interact with the user without Gui?

Reply:
Possibly, and I've seen this in a lot of other applications. Yet, I'm not sure if this is what you want. Just wanted to check, since it's the only logging line I've seen through all of this PR.

Reply:
THis only starts the game, and it's the only real useful info I thought of for the server at the moment. If you have any other idea on what I want to log feel free to open another comment :)